doc/ompl_interface/ompl_interface_tutorial.rst

```diff
@@ -65,6 +65,9 @@ Several planners that are part of the OMPL planning library are capable of optim
 * SPARS
 * SPARS2
 * Transition-based RRT (T-RRT)
+* AIT*
+* BIT*
```

Comment:
A few lines below, it is saying that BIT* is not implemented:
```
* Batch Informed Trees (BIT*)
```
I think that line should be removed in this PR.

Reply:
Absolutely. I missed that.